### ISSUE TITLE
x/build/cmd/coordinator: triggering slowbots is somewhat confusing

### ISSUE THREAD

**Issue by rolandshoemaker** [golang/go#59911]
Currently, the `TRY=` comment needs to happen at the same time as the Run-TryBot+1 vote, otherwise it is silently ignored, which can be incredibly confusing (at least to me), and requires a not insignificant number of actions to resolve (deleting votes, re-commenting and voting, etc).

It would seem somewhat preferable to simply use the last `TRY=` comment on the change to determine slowbots (perhaps this requires retrieving more information from gerrit than the coordinator currently has?). Reseting the selected slowbows would require an empty `TRY=` if you wanted to disable them, but overall that seems less confusing (at least to me) to the current behavior.

Given the rapid approach of LUCI, this may not be particularly important, but it may help trigger something in the back on my mind next time I forget how to do this.

cc @golang/release @bcmills

**Comment by dmitshur** [comment]
> It would seem somewhat preferable to simply use the last `TRY=` comment on the change to determine slowbots (perhaps this requires retrieving more information from gerrit than the coordinator currently has?).

Yeah. The current suboptimal behavior is due to an implementation shortcut, see second sentence in the [relevant comment](https://cs.opensource.google/go/x/build/+/master:maintner/maintnerd/maintapi/api.go;l=131-133;drc=a4dfb2681a32a893a4389637f244cc0137cea57d). It was meant to be improved at some point so that "TRY= comments are ignored if they're not on the same comment that started the TryBots" could be deleted from https://go.dev/wiki/SlowBots#pitfalls.

At this point, as you said, LUCI's implementation of opt-in trybots via a UI will likely supersede the `TRY=` comments and resolve this particular pitfall.
